news/5216.removal.rst

```diff
@@ -0,0 +1 @@
+Remove the ``pip search`` command as it is basically a no-op with PyPI.
```

Comment:
It’s not technically a no-op, the command “works” by sending a request and receive an error in response 😆 I’d say *it is no longer working* instead. I would also mention the command was never standardised and common usages of it can be replaced with `pip index`.

Reply:
Thanks, updated.